Terminal.Gui/ConsoleDrivers/WindowsDriver.cs

```diff
@@ -1695,7 +1695,13 @@ private MouseFlags ProcessButtonClick (WindowsConsole.MouseEventRecord mouseEven
 
     private async Task ProcessButtonDoubleClickedAsync ()
     {
-        await Task.Delay (300);
+        // Only delay if there's not a view wanting continuous button presses
+        if (Application.WantContinuousButtonPressedView is null)
+        {
+            // QUESTION: Why 300ms?
+            await Task.Delay (300);
+        }
```

Comment:
I think isn't awaiting at all and exit if `WantContinuousButtonPressedView` isn't null another button pressed that could chain on a double click or triple click will always sending button pressed.